include/vcpkg/base/messages.h

```diff
@@ -412,6 +412,32 @@ namespace vcpkg
             this->print(c, msg::format(m, args...).append_raw('\n'));
         }
 
+        void println_warning(const LocalizedString& s);
```

Comment:
I'm weakly against this: output redirection and formatting the error prefix on are separate concerns and I don't think `sink.println(msg::format_error(msgFoo))` is that onerous. But I am on record as not being a fan of any "format wrappers" so we should get other maintainers' thoughts.